libkbfs/journal_server.go

```diff
@@ -303,12 +291,19 @@ func (j *JournalServer) Disable(ctx context.Context, tlfID TlfID) (
 			"are any dirty blocks outstanding", tlfID)
 	}
 
-	tlfJournal.shutdown()
-
-	j.log.CDebugf(ctx, "Disabled journal for %s", tlfID)
+	// Disable the journal.  Note that we don't bother deleting the
```

Comment:
Probably not needed for this PR, but I was thinking about a related problem, where someone calls `getTlfJournal`, gets `nil` back, that goroutine gets switched out, then `Enable` is called on that TLF, then the first goroutine switches back, and it still hits the server even though it happens after `Enable` is called.

So I'm wondering if we need `getTlfJournal` to lazy-create `tlfJournal`s in a disabled state. But I can't think of a concrete problem this solves. And I guess even if we do this, someone could call `getTlfJournal`, get a disabled one back, try to use it and get the error`, and then get switched out while the TLF is enabled.

I just want to double-check if you think this isn't a problem.